app/javascript/components/lists/ListEntry.js

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import ListGroupButtons from "./ListGroupButtons"
+
+const ListEntry = (props) => {
+  const list = props.list;
+  const activeClass = props.activeClass;
+  const csrf = props.csrf;
+
+  return (
+    <React.Fragment>
+      <a key={list.id} className={"list-group-item list-group-item-action d-flex " + activeClass} id={"list-" + list.id + "-list"} data-bs-toggle="list" href={"#list-" + list.id} role="tab" aria-controls={"list-" + list.id}>
```

Comment:
Now that this is its own React component, it would be good to clean up the interpolation into variables that are interpolated. It would decrease the length of the line, decrease the interpolation, and remove the duplication.

Reply:
Will extrapolate today. Good feedback!